.prettierrc

```diff
@@ -11,7 +11,7 @@
   "quoteProps": "as-needed",
   "requirePragma": false,
   "semi": true,
-  "singleQuote": false,
+  "singleQuote": true,
```

Comment:
I recommend you don't modify the .prettierrc file. usually in a team these settings are shared by the whole dev team so it is good practice to leave them alone. (unless you are the person setting up the project)